guides/assets/stylesrc/_main.scss

```diff
@@ -699,6 +699,9 @@ body.guide {
       border-radius: $base-border-radius;
       font-size: 1rem;
       padding: 0.5em 1em;
+      // @include media('>desktop') {
+      //   padding: 0.5em 0.65em;
+      // }
```

Comment:
Can we remove the commented code?

Reply:
Sorry. That was an oversight. I meant to remove it.